locales-pending/premium.ftl

```diff
@@ -362,6 +362,7 @@ premium-upsell-dialog-title = Choose the level of protection that’s right for
 guided-resolution-flow-exit = Return to dashboard
 guided-resolution-flow-back-arrow = Go to previous step
 guided-resolution-flow-next-arrow = Go to next step
+guided-resoultion-flow-step-navigation-label = Guided steps
```

Comment:
```suggestion
guided-resolution-flow-step-navigation-label = Guided steps
```

Reply:
Updated in [d88cd80](https://github.com/mozilla/blurts-server/pull/3538/commits/d88cd80f0d22b37277021d85de6a1fe1dee7c57c).